src/quantities.js

```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
     "<rpm>"   :[["rpm"], 2.0*Math.PI / 60.0, "angular_velocity", ["<radian>"], ["<second>"]],
 
     /* memory */
```

Comment:
Could you replace `memory` by `information` everywhere in the file to keep the library on par with _Ruby-units_?